audition/distance_from_best.py

```diff
@@ -33,11 +33,23 @@ def _create(self):
             train_end_time timestamp,
             metric text,
             parameter text,
-            raw_value float,
-            below_best float,
-            below_best_next_time float
+            dist_from_abs_worst float,
+            dist_from_best_case float,
+            dist_from_best_case_next_time float
         )'''.format(self.distance_table))
 
+    def _metric_value_order(self, metric):
+        if greater_is_better(metric):
+            return 'desc'
+        else:
+            return 'asc'
+
+    def _abs_worst(self, metric):
```

Comment:
It's not necessarily true that metrics are bound by 0 and 1, so I don't think this will be general enough in practice.